plaid/investment_transactions.go

```diff
@@ -0,0 +1,99 @@
+package plaid
+
+import (
+	"encoding/json"
+	"errors"
+)
+
+type InvestmentTransaction struct {
+	InvestmentTransactionID string `json:"investment_transaction_id"`
+	AccountID               string `json:"account_id"`
+	SecurityID              string `json:"security_id"`
+	CancelTransactionID     string `json:"cancel_transaction_id"`
+
+	Date                   string  `json:"date"`
+	Name                   string  `json:"name"`
+	Quantity               float64 `json:"quantity"`
+	Amount                 float64 `json:"amount"`
+	Price                  float64 `json:"price"`
+	Fees                   float64 `json:"fees"`
+	Class                  string  `json:"class"`
+	ISOCurrencyCode        string  `json:"iso_currency_code"`
+	UnofficialCurrencyCode string  `json:"unofficial_currency_code"`
+}
+
+type GetInvestmentTransactionsResponse struct {
+	APIResponse
+	Item                        Item                    `json:"item"`
+	Accounts                    []Account               `json:"accounts"`
+	InvestmentTransactions      []InvestmentTransaction `json:"investment_transactions"`
+	Securities                  []Security              `json:"securities"`
+	TotalInvestmentTransactions int                     `json:"total_investment_transactions"`
+}
+
+type GetInvestmentTransactionsOptions struct {
+	StartDate  string
```

Comment:
this + `EndDate` shouldn't be in the options struct

Reply:
oh wait - nvm this is the param object passed in, ignore what I said

Reply:
It looks like `transactions` follows the same format: https://github.com/plaid/plaid-go/blob/master/plaid/transactions.go#L75
Alternatively we could have them pass it in as arguments to `GetInvestmentsTransactions` -- thoughts?

Reply:
If we're going to error when options.StartDate/options.EndDate are `""`, I'm fine with making them explicit arguments for the function. I'm also fine with being consistent with `GetTransactionsWithOptions`

Reply:
I think we should mimic the other methods in this lib – so matching `GetTransactionsWithOptions` is the best direction IMO

Reply:
Okay. In that case I'll be leaving `StartDate` and `EndDate` in the `GetInvestmentTransactionsOptions` struct